src/main/java/dev/dokan/dokan_java/constants/dokany/MountOption.java

```diff
@@ -20,20 +20,21 @@
     CURRENT_SESSION(128, "Mount the drive on current session only."),
     FILELOCK_USER_MODE(256, "Enable Lockfile/Unlockfile operations. Otherwise Dokan will take care of it.");
 
-    private final int mask;
+    private final int maskValue;
```

Comment:
The origin of the name was not so much the idea that the value holds a mask, but is used in a mask. MaskValue is the value, which is applied to the mask.

Reply:
Ahh, from this perspective it makes more sense^^ Maybe it can be stressed out in a way, that it represents exactly one single value? maskValue sounds like "value of the mask". Right now, i don't have a good suggestion.

But this won't be something, which blocks the PR.

Reply:
@infeo and I agreed to replace `maskValue` with `maskingValue` in those cases.
Fixed with 8ef36a92aeafb39610c0e5e4b2872e96d50a4f2f